crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -4507,11 +4627,14 @@ impl<'db> TypeInferenceBuilder<'db> {
                             let eq_result = self.infer_binary_type_comparison(
                                 Type::Tuple(lhs),
                                 ast::CmpOp::Eq,
-                                *ty,
+                                *ty, range
                             ).expect("infer_binary_type_comparison should never return None for `CmpOp::Eq`");
```

Comment:
```suggestion
                                *ty,
                                range,
                            ).expect("infer_binary_type_comparison should never return None for `CmpOp::Eq`");
```

Reply:
Hmm thanks. `infer` and `types.rs` are a bit much for Rustfmt.